File: backend/src/main/java/com/pCarpet/dto/EmployeeDTO.java
package com.pCarpet.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class EmployeeDTO {
	public int id;			//chiave della tabella employee
	private int idUser;		//id che identifica l'employee nella tabella user
	private int idTask;
	private String name;
	private int idBusinessOwner;	//id del business owner a cui fa riferimento il dipendente
}

File: frontend/src/dto/EmployeeDTO.ts
export class EmployeeDTO{
    id: number;
    idUser: number;
    idBusinessOwner: number;
    name: string;
    taskId: number;

    constructor(id: number, idUser: number, idBusinessOwner: number, name: string, taskId: number){
        this.id = id;
        this.idUser = idUser;
        this.idBusinessOwner = idBusinessOwner;
        this.name = name;
        this.taskId = taskId;

        
    }

    public setIdUser(id: number){
        this.idUser = id;
    }

    public setName(name: string){
        this.name = name;
    }

    public setIdTask(taskId: number): void{
        this.taskId = taskId;
    }

}